packs/equipment-effects/effect-prey-mutagen-lesser.json

```diff
@@ -0,0 +1,61 @@
+{
+    "_id": "HLn9nQY4SAvyhaom",
+    "img": "systems/pf2e/icons/equipment/alchemical-items/alchemical-elixirs/prey-mutagen.webp",
+    "name": "Effect: Prey Mutagen (Lesser)",
```

Comment:
Is there a way to condense these three effects? Maybe store the AC bonus value with AE likes then the action can pull its value from that data instead of using AdjustModifier?

Reply:
The problem is the duration. The lesser one is 1 minute, the moderate is 10 minutes, and the rest are 1 hour.